### DISCUSSION TITLE
Visibility of lead-organisations (opportunity-level)

### DISCUSSION THREAD

**Discussion by FloriSpoelstra** [alkem-io/alkemio#374] [Ideas] [answered]
It would be great to be able to see the logo or name of the opportunity leader (organisation) on the opportunities overview page. In that way it is easy to see if an opportunity is initiated by the government, the market or a science party.

**Comment by techsmyth** [comment]
Yes indeed this needs attention. Question for me is what should be the associations:
- there are members (users)
- there are also groups (which could be teams)
- there can be organisations
So if an Opportunity is being worked on by a team (like at a hackathon), then it maybe that we want both the ability to tags groups for visibility on the opportunity page as well as organisations. 

Thoughts? This may need a deeper discussion, as it also cascades through to challenge / project as well so that we get one consistent approach.

**Reply by FloriSpoelstra** [reply]
I have not yet explored the whole "groups" thing, but it sounds like a good plan to also allow them to be visible on the Challenge or Opportunities "card" as "lead party". I am not sure about individual member though. Although it could be interesting for people who are using the platform privately and not as part of an organisation. I don't think there is any harm in allowing al three.

**Comment by techsmyth** [comment, marked as answer]
Looking at this again it also occurs to me that we also need to decided is we want "lead organisations" also at the opportunity level - that is currently not there.

**Reply by FloriSpoelstra** [reply]
I would really like if it is straight-up visible who is involved in an opportunity. It would add value to see if it is a single organisation initiative or a coalition working on something, and if it is gov-only or multiple helix. 

On opportunity level I would even prefer to have the organisations visible on the opportunity card, in stead of the tags. Especially when the "[filter opportunities by present tags](https://github.com/alkem-io/client-web/issues/949)" function is implemented.

**Reply by techsmyth** [reply]
Hi Flori,

The filter opportunities item is actually in progress, it should make the next deployment :) Exact form being worked, but it will go beyond just tags to also filter on context fields such as tagline etc. 

Also re the opportunity + who is involved, I would like to probably pick this up with the Community Page epic: https://github.com/alkem-io/alkemio/issues/393 - feel free to add more details to that directly.

Cheers, Neil
